components/Common/ShellBox/index.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import ShellBox from '.';
+
+export default {
```

Comment:
Don't forget when refactoring to follow the Storybook example (default export goes after stories)